src/NUnitEngine/nunit.engine/Services/ExtensionService.cs

```diff
@@ -523,28 +533,33 @@ internal void FindExtensionsInAssembly(ExtensionAssembly assembly)
         /// </summary>
         /// <param name="runnerAsm">The executing runner</param>
         /// <param name="extensionAsm">The extension we are attempting to load</param>
-        internal static void ValidateTargetFramework(Assembly runnerAsm, ExtensionAssembly extensionAsm)
+        internal static bool ValidateTargetFramework(Assembly runnerAsm, ExtensionAssembly extensionAsm)
         {
             if (runnerAsm == null)
-                return;
+                return true;
 
             var extHelper = new TargetFrameworkHelper(extensionAsm.FilePath);
             var runnerHelper = new TargetFrameworkHelper(runnerAsm.Location);
             if (runnerHelper.FrameworkName?.StartsWith(".NETStandard") == true)
             {
-                throw new NUnitEngineException("Test runners must target .NET Core or .NET Framework, not .NET Standard");
+                log.Warning($"{runnerAsm.FullName} test runner must target .NET Core or .NET Framework, not .NET Standard");
```

Comment:
This one should still be an exception I think - this is looking at the framework of the runner. It's probably more of a sanity check than anything else however.